src/main/webapp/src/app/location/location.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Output, EventEmitter, ViewChild, ElementRef } from '@angular/core';
+import { LocationService } from '../location.service';
+import { Location } from '../location';
+import { string } from 'prop-types';
+
+@Component({
+  selector: 'app-location',
+  templateUrl: './location.component.html',
+  styleUrls: ['./location.component.css']
+})
+export class LocationComponent{
+
+  @Output() locationStored: EventEmitter<void> = new EventEmitter()
```

Comment:
I am unsure on what event emitter does in here, is it used to track onClick() event to set a new location?

Reply:
Good Catch. I think I need to remove it as I no longer need it. I made the event emitter initially to emit once the location had been stored on the client side. I wanted to use it to update the other components.